### PR TITLE
Device base testx

### PR DESCRIPTION
**Pull request by kvish**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@kvish__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

File: DeviceAdministration/Infrastructure.UnitTests/Simulator.WebJob/DeviceBaseTests.cs [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+            _transportFactory = new Mock<ITransportFactory>();
+            _telemetryFactoryMock = new Mock<ITelemetryFactory>();
+            _configurationProviderMock = new Mock<IConfigurationProvider>();
+            _transport = new Mock<ITransport>();
+            deviceBase = new DeviceBase(_loggerMock.Object, _transportFactory.Object, _telemetryFactoryMock.Object,
+                _configurationProviderMock.Object);
+        }
+
+        private readonly Mock<ILogger> _loggerMock;
+        private readonly Mock<ITransportFactory> _transportFactory;
```
Comment (handeadiguzel) [inline DeviceAdministration/Infrastructure.UnitTests/Simulator.WebJob/DeviceBaseTests.cs]: I usually place class variables before the constructor.

File: DeviceAdministration/Infrastructure.UnitTests/Simulator.WebJob/DeviceBaseTests.cs [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+using Microsoft.Azure.Devices.Applications.RemoteMonitoring.Simulator.WebJob.SimulatorCore.Transport;
+using Microsoft.Azure.Devices.Applications.RemoteMonitoring.Simulator.WebJob.SimulatorCore.Transport.Factory;
+using Moq;
+using Xunit;
+
+namespace Microsoft.Azure.Devices.Applications.RemoteMonitoring.DeviceAdmin.Infrastructure.UnitTests.Simulator.WebJob
+{
+    public class DeviceBaseTests
+    {
+        public DeviceBaseTests()
```
Comment (handeadiguzel) [inline DeviceAdministration/Infrastructure.UnitTests/Simulator.WebJob/DeviceBaseTests.cs]: test this method too? GetDeviceInfo()

Comment (kvish) [inline DeviceAdministration/Infrastructure.UnitTests/Simulator.WebJob/DeviceBaseTests.cs]: this method is being tested

**Comment by Inspiravetion** [comment]
lgtm

**Pull request merged**